facebook/__init__.py

```diff
@@ -200,18 +200,24 @@ def delete_request(self, user_id, request_id):
 
     def put_photo(self, image, album_path="me/photos", **kwargs):
         """
-        Upload an image using multipart/form-data.
+        Upload an image using image URL or multipart/form-data.
 
-        image - A file object representing the image to be uploaded.
+        image - An image URL or a file object representing the image to be uploaded.
```

Comment:
This line is [too long](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).